subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalBuildIntegrationTest.groovy

```diff
@@ -1256,6 +1256,9 @@ task generate(type: TransformerTask) {
         project.ext.inputDirs.split(',').each { inputs.dir(it) }
     }
 '''
+        executer.beforeExecute {
+            expectDeprecationWarning()
+        }
```

Comment:
Why not declare an output for this task instead?

Reply:
Done.